x/reward/spec/03_state_transitions.md

```diff
@@ -0,0 +1,63 @@
+<!--
+order: 3
+-->
+
+# State Transitions
+
+This document describes the state transition operations pertaining to:
+
+1. [Reward Programs](./03_state_transitions.md#reward-programs)
+2. [Reward Claims](./03_state_transitions.md#reward-claims)
+
+<!-- TOC 2 2 -->
```

Comment:
Need to `make update-tocs`.

Reply:
I think this was covered in the conversation below.